jujugui/static/gui/src/app/components/entity-content-readme/test-entity-content-readme.js

```diff
@@ -0,0 +1,126 @@
+/*
+This file is part of the Juju GUI, which lets users view and manage Juju
+environments within a graphical interface (https://launchpad.net/juju-gui).
+Copyright (C) 2015 Canonical Ltd.
+
+This program is free software: you can redistribute it and/or modify it under
+the terms of the GNU Affero General Public License version 3, as published by
+the Free Software Foundation.
+
+This program is distributed in the hope that it will be useful, but WITHOUT
+ANY WARRANTY; without even the implied warranties of MERCHANTABILITY,
+SATISFACTORY QUALITY, or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Affero
+General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License along
+with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+'use strict';
+
+var juju = {components: {}};
+var testUtils = React.addons.TestUtils;
+
+describe('EntityContentReadme', function() {
+  var mockEntity;
+
+  beforeAll(function(done) {
+    // By loading this file it adds the component to the juju components.
+    YUI().use('entity-content-readme', function() { done(); });
+  });
+
+  beforeEach(function() {
+    mockEntity = makeEntity();
+  });
+
+  afterEach(function() {
+    mockEntity = undefined;
+  });
+
+  function makeEntity(files=[]) {
```

Comment:
This `makeEntity` function is showing up in a lot of test files. Seems like it's a prime candidate to refactor into a shared location.